docs/release-notes.md

```diff
@@ -32,6 +32,21 @@ updates as well.
   2.9.10 has an infinite loop in a certain end-of-file situation." Fix
   by updating a dependency in documentation generation.
 
+### Zephyr-Rtos compatibility
+- for compatibility with zephyr 2.2.0 need to revert following commits:  
+    f4a9a9d  
+    470e2f3 (conflict: accept current changes)  
+    1084100  
+    41c5d50  
+    fefdea20  
+    8ecb89b  
+
+- zephyr-rtos is using mcuboot exact version reference in its manifest. You
```

Comment:
What you're trying to say by "exact version reference"  is linking to a specific commit sha?

Reply:
I mean https://github.com/zephyrproject-rtos/zephyr/blob/master/west.yml#L85-L86

I can rewrite:
zephyr-rtos references (by git SHA or git tag  )a compatible mcuboot version in its manifest